projects/stretch_ovmm/configs/agent/floorplanner_eval.yaml

```diff
@@ -63,8 +63,8 @@ AGENT:
       arm_joint_mask: [0, 0, 0, 0, 0, 0, 1] # the arm joints that the policy can control
       max_displacement: 0.25                     # used when training the policy
       max_turn_degrees: 30.0
-      min_turn_degrees: 5.0
-      min_displacement: 0.1
+      min_turn_degrees: 0.0
```

Comment:
Why should these be zero? Wont that cause issues?

Reply:
I think we were seeing the agent moving in real world. @ykarmesh do you think this would still be required with the new policy that makes bigger changes to tilt?


Maybe we can add a TODO here and remove it the next time we test

Reply:
Fine with removing this and adding again if the gaze skill doesnt work again

Reply:
Removed